README.md

```diff
@@ -262,7 +262,87 @@ export default NuxtAuthHandler({
 })
 ```
 
-Note that there's way more options inside the `nextAuth.options` object, see [here](https://next-auth.js.org/configuration/options#options) for all available options.
+Note: The above credential-provider example is taken over in part from the [NextAuth.js credentials example](https://next-auth.js.org/configuration/providers/credentials). It is _not_ considered safe for production usage and you would need to adapt it further, e.g., by calling another service that provides authentication, such as [strapi](#example-with-a-custom-strapi-jwt-provider)
+
+##### Example with a custom Strapi JWT provider 
+
+This section gives an example of how the `NuxtAuthHandler` can be configured to use Strapi JWTs for authentication via the `CredentialsProvider` provider.
+
+Ther are five places to configure `nuxt-auth` to work with Strapi:
+- `STRAPI_BASE_URL` in `.env`: Add the Strapi environment variable to your .env file.
+- [`runtimeConfig.STRAPI_BASE_URL`-key in `nuxt.config.ts`](#nuxtconfigts): Add the Strapi base url env variable. 
+- [`runtimeConfig.STRAPI_API_TOKEN_SECRET`-key in `nuxt.config.ts`](#nuxtconfigts): Optional: Allows server-to-server/machine-to-machine/headless usage through Strapi API Tokens. 
+- [`auth`-key in `nuxt.config.ts`](#nuxtconfigts): Configure the module itself, e.g., where the auth-endpoints are, what origin the app is deployed to, ...
+- [NuxtAuthHandler](#nuxtauthhandler): Configure the authentication behavior, e.g., what authentication providers to use
+
+For a production deployment, you will have to at least set the:
+- `STRAPI_BASE_URL` Strapi base URL for all API endpoints by default http://localhost:1337 
+- `STRAPI_API_TOKEN_SECRET` optional, but required for server-to-server and/or headless usage.
+
+1. In your .env file add the following lines:
```

Comment:
```suggestion
1. Create a `.env` file with the following lines:
```

Reply:
I'm assuming people using the readme already have set-up their `.env` file for production.
In the readme, there is an existing section about going into production regarding the `.env` file.
I recommend taking over my runtimeConfig() in all your examples and put all your origin/secret/basePath into it and pull the docs straight regarding the `.env` file usage.
This needs to be more apparent in its body content, in the actual examples you should only talk about 'adding' additional environment variables. 

**E.G:**
For proper workflow, go with the following: 
From your (docker or pm2) .env file -> nuxt.config -> put them in public or runtimeConfig() everywhere else. 
Don't spam process.env in nuxt. The initial .env should feed the nuxt.config environment and use that instead, the end.

**GitHub provider**
    GithubProvider.default({
      clientId: 'your-client-id',
      clientSecret: 'your-client-secret'
    }),

should become
  **secret: config.NUXT_SECRET,**
  providers: [
    GithubProvider.default({
      clientId: **config.GITHUB_CLIENT_ID,**
      clientSecret: **config.GITHUB_CLIENT_SECRET**
    }),
    
PS: The secret in the Github provider example is missing, this GitHub example goes nowhere other than play testing it with nuxt-auth in its current shape.

I can agree that new developers would find it easier that reading the actual value is nice, but above DX common sense and good practice comes first, so this should be reflected professionally. I modified my code back to using the runtimeConfig and not the process.env spam, as this is currently the best practice regarding '.env' file usage universally within Nuxt. What do you think?